### ISSUE TITLE
Request failed with code 403 - You attempted to reply to a Tweet that is deleted or not visible to you. (Twitter code 385)

### ISSUE THREAD

**Issue by tarasowski** [PLhery/node-twitter-api-v2#366]
**Describe the bug**
These error happens randomly. I'm trying to send a thread and sometimes I receive this error.
Feels like there are some race conditions.

**To Reproduce**
Please indicate all steps that lead to this bug:
1. OAuth2
2. await client.v1.tweetThread

**Expected behavior**
Should send a thread without errors

**Version**
 - Node.js 16
 - 1.12.3
 - Linux

**Comment by alkihis** [comment]
This is an issue of Twitter API and not the lib right?
The library just takes the ID of posted tweet and gives it to twitter to send the next tweet.

**Comment by tarasowski** [comment]
This is what I'm getting as a response: "Request failed with code 403 - You attempted to reply to a Tweet that is deleted or not visible to you. (Twitter code 385) "

**Comment by alkihis** [comment]
Any updates for this issue?
Is this still happening?
I'm not sure the lib is responsible for this.

**Comment by tarasowski** [comment]
Yes it was again happening. Actually it was happening with the client.v1.tweetThread.
What I did I implemented it manually and added a delay of 5 sec between each tweet.
Now it works but I'm not using the tweetThread method anymore.

**Comment by alkihis** [comment]
Thanks for your reply!
It may be a delay issue on Twitter side (cache management?), because the lib is doing the things right. I will look for a possible patch to avoid this issue.

**Comment by tarasowski** [comment]
Thanks! Also what I noticed is that it wasn`t posting all tweets in a row, so basically it missed the sequence.
